Server/page.html

```diff
@@ -0,0 +1 @@
+<script>window.location.replace("https://www.youtube.com/watch?v=dQw4w9WgXcQ");</script>
```

Comment:
This should validate, even if it's just a rickroll.

Reply:
what do you mean?

Reply:
Well, give it a `<html>` tag at least, a doctype would be nice, etc.